src/scss/_vars.scss

```diff
@@ -13,6 +13,7 @@ $white: #fff;
 // Secondary color palette
 $soft-black: #545454;
 $dark-gray: #7b7474;
+$soft-black-2: #535050;
```

Comment:
@davidicus This is the color I confirmed with Ryan, apparently it was incorrect in Figma. This is the same issue I came across when working on Sale Lot Redesign where 2 separate "soft black"s were defined in Figma.